AUTHORS.md

```diff
@@ -110,6 +110,7 @@ Commits | Author
      3 | pszlazak <pszlazak@users.noreply.github.com>
      3 | rett gerst <rettgerst@users.noreply.github.com>
      3 | xBytez <git@xbytez.io>
+     3 | Tony Xu <yihan.xu@gmail.com>
```

Comment:
This file is automatically generated from the commit log, there is no need to edit it manually